pkg/agentfs/examples/node.Dockerfile

```diff
@@ -6,20 +6,21 @@ WORKDIR /app
 
 RUN npm install -g pnpm@9.7.0
 
-# throw away build stage to reduce size of final image
+# Throw away build stage to reduce size of final image
 FROM base AS build
 
 RUN apt-get update -qq && apt-get install --no-install-recommends -y ca-certificates
 COPY --link . .
 
 RUN pnpm install --frozen-lockfile
-RUN npm run build
+RUN pnpm run build
```

Comment:
you may also want to detect yarn (yarn.lock) or npm (package-lock.json) projects, and rename the base one here to `node.pnpm.Dockerfile`